himbaechel/uarch/gowin/pack.cc

```diff
@@ -1798,6 +1823,15 @@ struct GowinPacker
             bsram_fix_blksel(ci, new_cells);
         }
 
+        // XXX
+        NetInfo *oce_net = ci->getPort(id_OCE);
+        if (oce_net == nullptr || oce_net->name == ctx->id("$PACKER_VCC") || oce_net->name == ctx->id("$PACKER_GND")) {
+            if (oce_net != nullptr) {
```

Comment:
why is this checking `oce_net == nullptr` and then `oce_net != nullptr` inside of it?

Reply:
Corrected.